pyproject.toml

```diff
@@ -1,33 +1,32 @@
 [tool.poetry]
 name = "py-irt"
-version = "0.4.13"
+version = "0.4.14"
 readme = "README.md"
 homepage = "https://github.com/nd-ball/py-irt/"
 description = "Bayesian IRT models in Python"
 authors = ["John P. Lalor <john.lalor@nd.edu>"]
 license = "MIT License"
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.12"
-numpy = ">=1.22.0"
-pandas = "^1.1"
-pyro-ppl = "^1.6.0"
-rich = "^9.3.0"
+python = ">=3.9,<3.12"
```

Comment:
Could you also update .github/workflows/pythonpackage.yml to exclude Python 3.8?